src/components/Header.js

```diff
@@ -1,45 +1,49 @@
 import React from 'react'
 import { Link } from 'react-router'
 
-const Header = () => {
-  const active = {
-    borderBottom: '5px solid #ff5e50',
-    fontWeight: 'bold',
-  }
+const isExplorerActive = location => {
```

Comment:
thoughts on combining the two truth conditions and using the new `includes` string method?

```
const isExplorerActive = location => (
  location.pathname === '/' || location.pathname.includes('explorer')
)
```